### PR TITLE
feat: support to send logs

### PR DESCRIPTION
**Pull request by kuisathaverat**
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/opentelemetry/job/log/OtelLogSenderRoot.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package io.jenkins.plugins.opentelemetry.job.log;
+
+/**
+ * Sender for logs send from the controller, pipeline flow control and pipeline steps ran in Jenkins controller.
+ * See https://github.com/jenkinsci/pipeline-cloudwatch-logs-plugin/blob/master/src/main/java/io/jenkins/plugins/pipeline_cloudwatch_logs/CloudWatchSender.java#L79
+ */
+final class OtelLogSenderRoot extends OtelLogSender {
```
Comment (cyrille-leclerc) [inline src/main/java/io/jenkins/plugins/opentelemetry/job/log/OtelLogSenderRoot.java:7]: I simplified just after our call removing OtelLogSenderRoot and OtelLogSenderFlowNode just keep one OtelLogSenderBuildListener, sorry for the mess

**Comment by cyrille-leclerc** [comment]
I stop touching the send-build-logs-to-otel branch so that we don't conflict. I'll resume once you have merged.

Please have a look at this proposal I just added:
- https://github.com/jenkinsci/opentelemetry-plugin/pull/306

File: pom.xml [hunk 2]
```diff
@@ -339,6 +340,17 @@
...
+            <groupId>org.elasticsearch.client</groupId>
+            <artifactId>elasticsearch-rest-high-level-client</artifactId>
+            <version>${elasticstack.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>net.sf.jopt-simple</groupId>
+                    <artifactId>jopt-simple</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```
Comment (cyrille-leclerc) [inline pom.xml:353]: Do you have in mind that "Apache HTTP Components" is packaged as a Jenkins Plugin? From what I remember of some Elasticsearch client jars, there is a dependency on Apache HTTP.

Comment (kuisathaverat) [reply pom.xml:353]: is a TODO to review if we can free weight from the Elasticsearch client

**Comment by kuisathaverat** [comment]
superseded by https://github.com/jenkinsci/opentelemetry-plugin/pull/307

**Pull request closed without merging**